ghproxy/ghcache/ghcache.go

```diff
@@ -156,6 +157,16 @@ type upstreamTransport struct {
 
 func (u upstreamTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	etag := req.Header.Get("if-none-match")
+
+	// get authorization header to convert to sha256
+	authHeader := req.Header.Get("Authorization")
+	hasher := sha256.New()
+	hasher.Write([]byte(authHeader))
+	authHeaderHash := string(hasher.Sum(nil))
+	if authHeader == "" {
+		logrus.Warnf("Couldn't retrieve 'Authorization' header, %s is the hash of an empty string", authHeaderHash)
```

Comment:
Warning level logs mean that an admin should take action. What action should they take here? If none, let's log at info

Reply:
I'd assume that we actually don't have unauthenticated requests to GitHub. If message is ever logged, there's something wrong with the setup. That's why I set it to warning level.